package/koreader/package

```diff
@@ -2,15 +2,15 @@
 pkgname=koreader
 pkgdesc="KOReader"
 url=https://github.com/koreader/koreader
-pkgver=2020.08.01-1
-timestamp=2020-09-19T03:30Z
+pkgver=2020.09-1
+timestamp=2020-09-25T01:42Z
 section=utils
 maintainer="raisjn <of.raisjn@gmail.com>"
 license=AGPL3
```

Comment:
The correct [SPDX identifier](https://spdx.org/licenses/) for this license is `AGPL-3.0-or-later`.